elements/power.lua

```diff
@@ -398,6 +415,10 @@ local function Enable(self)
 			element:SetStatusBarTexture([[Interface\TargetingFrame\UI-StatusBar]])
 		end
 
+		if(element.colorPowerAtlas) then
+			element.__texture = element:GetStatusBarTexture():GetTexture()
```

Comment:
`element.__texture = element.__texture or element:GetStatusBarTexture():GetTexture()` might be a better idea in case it's not the initial `Enable` call because the bar might be using an atlas from before. So trying to recover it from `element.__texture` is safer, and users might set it themselves to avoid any inconsistencies as well.